integration/tests/economy/test_economics.py

```diff
@@ -637,7 +637,8 @@ def test_deploy_contract_to_exist_unpayed(
         )
         get_gas_used_percent(w3_client, contract_deploy_tx)
 
-    @pytest.mark.timeout(960)
+    @pytest.mark.slow
+    @pytest.mark.timeout(16 * Time.MINUTE)
```

Comment:
Did you check this test? After changes it passed?

Reply:
Yes, it's passing